src/CodeStyle/VisualBasic/CodeFixes/Microsoft.CodeAnalysis.VisualBasic.CodeStyle.Fixes.vbproj

```diff
@@ -5,7 +5,7 @@
     <Platform Condition="'$(Platform)' == ''">AnyCPU</Platform>
     <PlatformTarget>AnyCPU</PlatformTarget>
     <OutputType>Library</OutputType>
-    <TargetFramework>netstandard1.3</TargetFramework>
+    <TargetFramework>net46</TargetFramework>
```

Comment:
Why?

Reply:
➡️ Discussion in https://github.com/dotnet/roslyn/pull/28873#discussion_r222879899; the change in this project file was specifically the result of the change in one of its dependencies.